ajax/libs/ant-design-vue/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "ant-design-vue",
+  "filename": "antd.min.js",
+  "version": "1.3.8",
```

Comment:
The `version` attr should not be included, as these files do not exist in this PR.